apps/fyler/src/handlers/audio_to_mp3.erl

```diff
@@ -16,23 +16,35 @@ category() ->
 
 run(File) -> run(File,[]).
 
-run(#file{tmp_path = Path, name = Name, dir = Dir, extension = Ext},_Opts) ->
+run(#file{tmp_path = Path, name = Name, dir = Dir, extension = Ext}, _Opts) ->
   Start = ulitos:timestamp(),
 
   NewName = if Ext =:= "mp3" 
-      -> Name++"_converted";
+      -> Name ++ "_converted";
       true -> Name
   end,
 
-  MP3 = filename:join(Dir,NewName++".mp3"),
+  MP3 = filename:join(Dir, NewName ++ ".mp3"),
 
-  Command = ?COMMAND(Path,MP3," -c:a libmp3lame -ac 2 -ar 48000 -ab 192k "),
+  Command = ?COMMAND(Path, MP3, " -c:a libmp3lame -ac 2 -ar 48000 -ab 192k "),
 
   ?D({"command",Command}),
-  Data = os:cmd(Command),
-  case filelib:wildcard("*.mp3",Dir) of
+  Data = exec(Command),
+  case filelib:wildcard("*.mp3", Dir) of
     [] -> {error,Data};
     _List ->
       Result = NewName++".mp3",
       {ok,#job_stats{time_spent = ulitos:timestamp() - Start, result_path = [list_to_binary(Result)]}}
```

Comment:
Line 37 is too long:       {ok,#job_stats{time_spent = ulitos:timestamp() - Start, result_path = [list_to_binary(Result)]}}.